libs/ledger-live-common/src/wallet-api/types.ts

```diff
@@ -61,3 +62,12 @@ export type ConvertToLiveTransaction<
   T extends WalletAPITransaction,
   U extends Transaction
 > = (tx: T) => Partial<U>;
+
+export interface DiscoverDB {
+  recentlyUsed: RecentlyUsed[];
+}
+
+export interface RecentlyUsed {
+  id: string;
+  usedAt: string;
+}
```

Comment:
You might want to use `type` instead of `interface` here to avoid unwanted merge by ts